be/src/vec/columns/column.h

```diff
@@ -478,11 +479,19 @@ class IColumn : public COW<IColumn> {
       * nan_direction_hint - see above.
       */
     virtual void get_permutation(bool reverse, size_t limit, int nan_direction_hint,
-                                 Permutation& res) const {
+                                 HybridSorter& sorter, Permutation& res) const {
         throw doris::Exception(ErrorCode::NOT_IMPLEMENTED_ERROR,
                                "get_permutation for " + get_name());
     }
 
+#ifdef BE_TEST
```

Comment:
if only define for test maybe not be column.h better, which may all file include column.h need to recompile